frontend/src/components/ForceScales/ForceScales.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import SliderControl from '../SliderControl/SliderControl.jsx';
+import styles from './ForceScales.css';
+
+/*
+    <SliderControl min='0' max='100' key={'thrust0'} indx={0} val={this.state.scales[0]} inv={this.state.inv[0]} rend={this.rendData.bind(this)} name={"Thruster 0"} />
+*/
+
+export default class ForceScale extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { scales: props.scales };
+
+        this.rendLeftScales = this.rendLeftScales.bind(this);
+        this.rendRightScales = this.rendRightScales.bind(this);
+        this.rendData = this.rendData.bind(this);
+    }
+
+    rendData(val, inv, key) {
+        //console.log(this.state.scales[key]);
+        let scalescpy = this.state.scales;
+        scalescpy[key] = val;
+        //scalescpy[key].invert = inv;
+        this.setState({
+                scales: scalescpy,
+            },
+            function() {
+                this.props.rend(this.state.scales);
+            }
+        );
+    }
```

Comment:
wait, so this for the sliders. So each slider, when data changes, are you resetting the state, but _also_ resetting the props and rerendering the component with by setting the props to the new value? I think you should only be setting the state here..

Reply:
From my understanding, I don't think that changes the original packet... the whole idea of using props.rend() is so that the parent gets the child's change of information.

I agree, doing both is a little vague though; I'm going to see if I can do this only through props.

I originally did just the state, and then wanted to make sure that it was updating so I converted the text values to be based off the props... and that's where it got left